release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -48,6 +48,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-10-conditional-updates"]
+=== Conditional updates
+
+{product-title} 4.10 adds support for consuming conditional update paths provided by OpenShift Update Service.
+Conditional update paths convey identified risks and the conditions under which those risks apply to clusters.
+The Administrator perspective on the web console only offers recommended upgrade paths for which the cluster does not match known risks.
+However, {product-title} 4.10 clusters using `oc` 4.10 or later can be used to display additional upgrade paths with associated risk information including supporting documentation references.
+The administrator may review the referenced materials and choose to perform the supported but no longer recommended upgrade.
```

Comment:
IBMSG always recommends avoiding "may". (I get nailed for that all the time, lol) 

Any strong opinions against separating "but no longer recommended" by commas? ie: 

The administrator may review the referenced materials and choose to perform the supported, but no longer recommended, upgrade.

It might help readability. Entirely up to you whether you change it.